.github/workflows/master.yml

```diff
@@ -60,9 +60,9 @@ jobs:
         export SPARK_HOME="$SPARK_CACHE_DIR/spark-$SPARK_VERSION-bin-hadoop2.7"
         ./dev/lint-python
         ./dev/pytest
-    # Push the results back to codecov
-    - name: Update Codecov
-      run: bash <(curl -s https://codecov.io/bash)
+    - uses: codecov/codecov-action@v1
+      with:
+        token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Does this step work fine for a PR from a forked repo?

[The doc](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets#using-encrypted-secrets-in-a-workflow) says:

> With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository.

Reply:
Thanks @harupy. I think PR triggers codecov against the main repo. I'll investigate more tomorrow..

Reply:
The secret doesn't seem to be passed to the runner.

<img width="516" alt="Screen Shot 2020-02-10 at 12 01 00" src="https://user-images.githubusercontent.com/17039389/74118058-3c60b000-4bfd-11ea-87ff-52387ae44de7.png">

Reply:
or is it just hidden?

Reply:
I should add a Github Actions secret for `CODECOV_TOKEN`; however, I currently don't have the admin permission. Once that's set, everything should work fine.